xbmc/cores/RetroPlayer/playback/ReversiblePlayback.cpp

```diff
@@ -134,11 +134,16 @@ std::string CReversiblePlayback::CreateSavestate(bool autosave)
 
   std::string caption = m_cheevos->GetRichPresenceEvaluation();
 
-  if (!m_autosavePath.empty())
+  if (autosave)
   {
-    std::unique_ptr<ISavestate> loadedSavestate = CSavestateDatabase::AllocateSavestate();
-    if (m_savestateDatabase->GetSavestate(m_autosavePath, *loadedSavestate))
-      label = loadedSavestate->Label();
+    if (!m_autosavePath.empty())
+    {
+      std::unique_ptr<ISavestate> loadedSavestate = CSavestateDatabase::AllocateSavestate();
+      if (m_savestateDatabase->GetSavestate(m_autosavePath, *loadedSavestate))
+        label = loadedSavestate->Label();
```

Comment:
The savestate object has a `Type()` method that can tell if it's an autosave. Should we use that?